core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-content-id.service.ts

```diff
@@ -45,10 +45,13 @@ export class TranslationTabActiveContentIdService {
   }
 
   setActiveContent(contentId: string, dataFormat: string): void {
-    var allContentIds = (
-      this._stateRecordedVoiceoversService.displayed.getAllContentIds());
-    if (allContentIds.indexOf(contentId) === -1) {
-      throw new Error('Invalid active content id: ' + contentId);
+    const displayStateRecordedVoiceovers = (
+      this._stateRecordedVoiceoversService.displayed);
+    if (displayStateRecordedVoiceovers !== null) {
```

Comment:
Throwing null is not best approach, as in protactor there is no rejection callback handler for this particular code. So, it may fail many suites in e2e.

Reply:
Lets not care about any tests for now. Is throwing error here the correct thing to do? Basically, does it conceptually make sense when `displayStateRecordedVoiceovers` is `null`?

Reply:
Removed, make no sense.